templates/deployments.yaml

```diff
@@ -106,6 +104,141 @@ spec:
+      imagePullSecrets:
+      - name: {{ .Release.Name }}-st2-license
+      {{- end }}
+      initContainers:
+      {{- if .Values.st2.packs.image.repository }}
+      # Merge packs and virtualenvs from st2actionrunner with those from the st2.packs image
+      # Can be used for chatops pack if system pack comes without it.
+      - name: st2-custom-packs
+        image: "{{ .Values.st2.packs.image.repository }}/{{ .Values.st2.packs.image.name }}:{{ .Values.st2.packs.image.tag }}"
+        imagePullPolicy: {{ .Values.st2.packs.image.pullPolicy | quote }}
+        volumeMounts:
+        - name: st2-packs-vol
+          mountPath: /opt/stackstorm/packs-shared
+        - name: st2-virtualenvs-vol
+          mountPath: /opt/stackstorm/virtualenvs-shared
+        command:
+          - 'sh'
+          - '-ec'
+          - |
+            /bin/cp -aR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
+            /bin/cp -aR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
+      # System packs
+      - name: st2-system-packs
+        image: "{{ template "imageRepository" . }}/st2actionrunner{{ template "enterpriseSuffix" . }}:{{ .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        volumeMounts:
+        - name: st2-packs-vol
+          mountPath: /opt/stackstorm/packs-shared
+        command:
+          - 'sh'
+          - '-ec'
+          - |
+            /bin/cp -aR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
+            /bin/cp -aR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
+      {{- end }}
+      containers:
+      - name: st2chatops
+        image: gcr.io/rapitt/st2chatops-debug
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: ST2_AUTH_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-st2-auth
+              key: username
+        - name: ST2_AUTH_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-st2-auth
+              key: username
+        envFrom:
+        - configMapRef:
+            name: {{ .Release.Name }}-chatops-configs
+        command:
+          - |
+              docker inspect -f {{.State.Status}} stackstorm-hubot
+              docker logs stackstorm-hubot
+        resources:
+          requests:
+            memory: "5Mi"
+            cpu: "5m"
+      volumes:
+        - name: st2-config-vol
+          mountPath: /etc/st2/st2.docker.conf
+          subPath: st2.docker.conf
+        {{- if .Values.enterprise.enabled }}
+        - name: st2-rbac-roles-vol
+          configMap:
+            name: {{ .Release.Name }}-st2-rbac-roles
+        - name: st2-rbac-assignments-vol
+          configMap:
+            name: {{ .Release.Name }}-st2-rbac-assignments
+        - name: st2-rbac-mappings-vol
+          configMap:
+            name: {{ .Release.Name }}-st2-rbac-mappings
+        {{- end }}
```

Comment:
All RBAC-related options can be safely removed, st2chatops component doesn't need it.